test/e2e/storage/persistent_volumes-local.go

```diff
@@ -1829,7 +1836,24 @@ func createStatefulSet(config *localTestConfig, ssReplicas int32, volumeCount in
 					TopologyKey: "kubernetes.io/hostname",
 				},
 			},
-		},
+		}
+	} else {
+		affinity.PodAffinity = &v1.PodAffinity{
+			RequiredDuringSchedulingIgnoredDuringExecution: []v1.PodAffinityTerm{
```

Comment:
can we reuse the PodAffinityTerm in both if else cases ?

Reply:
sure, done